sirepo/raydata/scan_monitor.py

```diff
@@ -214,11 +214,20 @@ def _filename_and_image(path):
                 ),
             )
 
-        def _paths(uid):
+        def _filename_and_json(path):
+            return PKDict(filename=str(path.basename), json=pkio.read_text(path))
```

Comment:
I think path.basename is already a string

Instead of read_text if you call pkjson.load_any then you can remove the JSON.parse in the GUI

Reply:
f9683c34b2bc41b6f042087a75e103166fba7053